CMakeLists.txt

```diff
@@ -75,18 +75,18 @@ if(UNIX OR MINGW)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 endif()
 
-include("cmake/mkldnn_compat.cmake")
```

Comment:
If full paths are used (`cmake/` prefix), including `.cmake` files works only within MKL-DNN, we can't include them from Open Image Denoise (which has MKL-DNN as a subproject) because CMake includes are unfortunately relative to the source directory, not the file that contains the include. Dropping full paths fixes this issue.

Reply:
Thanks, I see. But we have a sub-project "test" in the repository:
https://github.com/intel/mkl-dnn/blob/master/tests/other/subproject/CMakeLists.txt

and it works fine even without this change. I'm not sure what might be the reason that we are not seeing the issues you described with our small sub-project...

Reply:
This is because OIDn [includes files from DNNL directly](https://github.com/OpenImageDenoise/oidn/blob/dabfd9c80101edae9d25a710160d12d6d963c591/CMakeLists.txt#L90-L94)

Reply:
What you describe is completely different from our case. MKL-DNN is the subproject of OIDN, not the other way around. But the issue is not related to subprojects. In OIDN we directly include some `.cmake` files from MKL-DNN, which fails if that file includes other files with full path.

Reply:
Closing the discussion as the changes [are not relevant anymore](https://github.com/intel/mkl-dnn/pull/572#issuecomment-544695471)